concordia/admin/actions.py

```diff
@@ -1,9 +1,25 @@
+import uuid
+
 from django.contrib import messages
 from django.utils.timezone import now
 
 from ..models import Asset, Transcription, TranscriptionStatus
 
 
+def anonymize_action(modeladmin, request, queryset):
+    count = queryset.count()
+    for user_account in queryset:
+        user_account.username = uuid.uuid4()
+        user_account.email = ""
+        user_account.password = ""
+        user_account.save()
```

Comment:
We should also clear `is_active` since we're resetting the email address as well:

```suggestion
        user_account.is_active = False
        user_account.save()
```